proposals/tools.md

```diff
@@ -0,0 +1,90 @@
+# OCI Runtime-Tools and Image-Tools Project Proposals
+
+## Abstract
+The below proposal contemplates creation of two tools projects (runtime-tools and image-tools.)
+These would be associated with the OCI specification projects (runtime-spec and image-spec) and serve as repositories for testing  tools. 
+
+## Background
+Testing tools have been organically developed by the OCI developer community.
+These include:
+* Runtime-spec test code in a OCITools repository
+* Image-spec test code in the image-spec repository
+
+On Monday, August 22nd, 2016, the OCI Certification Program WG designated Chris Aniszczyk and Rob Dolin to draft a proposal for formalizing approval of tools.
+
+On Wednesday, August 24th, 2016, the OCI Developer Community (TDC) met and recommended establishing a tools repository corresponding to each of the spec repositories.  
+
+## Proposal
+With repositories under the http://github.com/OpenContainers organization:
+* Rename the OCITools repository as runtime-tools
+* Create a new repository named image-tools and populate this with the files from: https://github.com/opencontainers/image-spec/tree/master/cmd/oci-image-tool
```

Comment:
In off-list discussion starting [here][1], I think, (and understand @RobDolinMS to agree) that we want to say that the repo names should end with `{component}-tools` (e.g. `runtime-tools`).  That way the tool maintainers can choose `oci-runtime-tools` (e.g. to match an installed command) if they like, or choose `runtime-tools` if there is no installed command to match, or if they (for whatever reason) don't want the repo name to match the installed command.

[1]: https://github.com/RobDolinMS/tob/commit/c1a42654c54946ea1427583c6bfb61ae6b0e1dd6#commitcomment-18803722